src/models/parallelCoordinates.js

```diff
@@ -12,6 +12,7 @@ nv.models.parallelCoordinates = function() {
         , height = null
         , x = d3.scale.ordinal()
         , y = {}
+        , undefinedValues = "undefined values"
```

Comment:
should probably be called "undefinedLabel" or something saying "Label" to match controlLables option which is used to change the name of control buttons.